### PR TITLE
Minor selection box state fixes

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
Two minor CSS fixes:

1. Focus state for radio buttons inside selection box will now be circular to match the natural shape of the radio button.
2. Focusing on a "selected" selection box will no longer remove the left border.

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-selection-box-state-fix](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-selection-box-state-fix)

File: src/app/public/modules/selection-box/selection-box.component.scss [hunk 1]
```diff
@@ -20,11 +20,9 @@
...
   &:focus {
     .sky-switch-control {
-      //default
-      outline: thin dotted;
-
-      //webkit
-      outline: -webkit-focus-ring-color auto 5px;
+      outline: none;
+      border: none;
+      box-shadow: inset 0 0 0 2px $sky-theme-modern-color-blue-74;
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/selection-box/selection-box.component.scss:25]: Modern color for default theme?

Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/selection-box/selection-box.component.scss]: Good catch. That should have only been applied to modern theme. The default-themed checkboxes should not be affected here. Fixed!

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-selection-box-state-fix](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-selection-box-state-fix)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**